contrib-kubernetes/README.md

```diff
@@ -0,0 +1,27 @@
+# `Kubernetes` Aliases and Utilities
+
+Enhances `Kubernetes` command line integration by providing aliases,
+functions and by exposing repository status information to prompts.
+
+## Aliases
+
+- `kb` is an alias for `kubectl`
+- `kba` is an alias for `ubectl apply`
```

Comment:
There's a missing `k` here (Looks like it was missing in the original as well).

Reply:
Yes.